config.default.toml

```diff
@@ -143,6 +145,14 @@ exec = [
   keys = [ "1", "2", "3", "4", "5", "6", "7", "8", "9" ]
   modifiers = [ "super", "shift", "control" ]
 
+	[controls.incWidthDiff]
+	keys = [ "l" ]
+	modifiers = [ "super" ]
+
+	[controls.decWidthDiff]
+	keys = [ "h" ]
+	modifiers = [ "super" ]
```

Comment:
Need to changes these as well and we should be good.

Reply:
Ok